docs/introduction.md

```diff
@@ -7,7 +7,7 @@ Writing security-conscious IAM Policies by hand can be very tedious and ineffici
 
  * Determined to make your best effort to give users and roles the least amount of privilege you need to perform your duties, you spend way too much time combing through the AWS IAM Documentation on [Actions, Resources, and Condition Keys for AWS Services][1].
  * Your team lead encourages you to build security into your IAM Policies for product quality, but eventually you get frustrated due to project deadlines.
- * You don't have an embedded security person on your team who can write those IAM policies for you, and there's no automated tool that will automagically sense the AWS API calls that you perform and then write them for you in a least-privilege manner.
+ * You don't have an embedded security person on your team who can write those IAM policies for you, and there's no automated tool that will [automagically sense the AWS API calls that you perform](https://github.com/princespaghetti/actionhero) and then write them for you in a least-privilege manner.
```

Comment:
I'll modify this sentence to reflect the changes in the README after merge so it is consistent.

Reply:
Ah my bad I forgot this part is duplicated in two files, sorry for the oversight.
See you fixed it in https://github.com/salesforce/policy_sentry/commit/cd49c1bd0fccc7eb9ea9961ae6dfe8186c2887a8 , thanks for taking care of that!